src/moa/moa_controllers/setup.py

```diff
@@ -21,7 +21,7 @@
     tests_require=['pytest'],
     entry_points={
         'console_scripts': [
-            'ack_to_can_node = moa_controllers.ackermann_to_motec:main'
+            'ack_to_can_node = moa_controllers.ackermann_to_can:main'
```

Comment:
need to re name file if you are changing this. Files still called ack_to_motec. the lay out for this line is:
```
'{name of command} = {name of pkg}.{name of file}:{name of function to call}'
```

Reply:
Yea I haven’t changed it yet because if I did it would remove the comments associated with the file. I will change it once I finish the comments on it.